packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/mfaoob/configuration.md

```diff
@@ -0,0 +1,31 @@
+## Enable authenticators for your org
+
+The direct authentication MFA OOB flow uses passwordless authentication, such as signing in with email or SMS. You need an authenticator enabled to use the direct authentication MFA OOB flow.
+
+> **Note:** Okta currently supports Okta Verify for the MFA OOB flow. <!-- Need to update this after the entire feature is rolled out and not limited to Okta Verify>
+
+1. Open the Admin Console for your org.
+1. Select **Security** > **Authenticators** to view the available authenticators.
+1. If the authenticator that you want to use isn't in the list:
+    * Click **Add authenticator**.
+    * Click **Add** on the authenticator tile, and then click **Add** in the next dialog.
+    * Select the **Enrollment** tab.
+    * Verify that the authenticator is set to either **Optional** or **Required** in the **Eligible authenticators** section of the Default Policy.
+1. If the authenticator that you want to use is set to **Disabled**, click **Edit** for the Default Policy.
+1. Select **Optional** from the dropdown box for the authenticator, and then click **Update Policy**.
+
+## Set up your authorization server
+
+To use the MFA OOB flow, both your client app and the [Okta authorization server](/docs/concepts/auth-servers/) that you are using with the app must have the MFA OOB grant type enabled in your Okta org.
+
```

Comment:
I actually think that the most common use case will be using direct auth with the org authorization server, in which case no configuration is needed because there are no custom policies for the org authorization server. I think we can either remove this section from all pages or we can clarify that IF DA is being used with a custom authorization server, the grant type should be enabled in the custom AS access policy with the steps you listed